modules/gateways/blockonomics/blockonomics.php

```diff
@@ -281,6 +266,24 @@ public function convertFiatToBlockonomicsCurrency($fiat_amount, $currency, $bloc
         return intval(1.0e8 * $fiat_amount / $price);
     }
 
+    /**
+     * Convert received btc percentage to invoice currency
+     *
+     * @param array $order
+     * @param string $percentPaid
+     * @return float converted value
+     */
+    public function convertPercentPaidToInvoiceCurrency($order, $percentPaid)
+    {
```

Comment:
Lets add code comments explaining motivation and convertToProcessing so that we can later understand this

Reply:
Added code comments in 9368f91